DependencyInjection/App/Presenters/AnotherPresenter.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace App\Presenters;
+
+use App;
+use Vendor;
+
+class AnotherPresenter extends Vendor\Presenters\SomePresenter
+{
+	/**
```

Comment:
pokud ano, tak o něm nevím. @vitkutny sám používá víceřádkové anotace i v inline kódu, což je za mě zufíl...

evidentně je kolem toho spíš zmatek, viz https://github.com/peckadesign/pdproject5/pull/1129#discussion_r126338322, takže bychom to měli sjednotit